wiki/Community/Forum/en.md

```diff
@@ -0,0 +1,71 @@
+---
+tags:
+  - forums
+  - subforum
+---
+
+# Forum
+
+*See also: [Internet forum](https://en.wikipedia.org/wiki/Internet_forum)*
+
+[The osu! **forums**](https://osu.ppy.sh/community/forums) are a place for conversations where messages are grouped by subjects, and no quick response is expected. The discussions are searchable, stored forever, and may be revisited at any time, which makes them different from the near-instant messages in [osu! chat](/wiki/Community/Internet_Relay_Chat). As a result, forum messages are longer, contain more information, and typically provide more details than chat messages would.
+
+The osu! forums provide [features](#features) similar to most internet forums: [BBCode](/wiki/BBCode) markup, polls, pins and subscriptions, private messages, and [moderation](#moderation).
+
+## Structure
+
+Each of the osu! forums contains a variety of topics, called **threads**, as well as smaller categories, or **subforums**. Most of the forums are English only, but there also exists a number of [language-specific](https://osu.ppy.sh/community/forums#forum-23) forums for local communities that are large enough.
+
+While some forums, such as [Mapping Discussion](https://osu.ppy.sh/community/forums/56), have specific requirements or guidelines, all of them follow [the community rules](/wiki/Rules).
+
+## Roles and user groups
+
+Some users may belong to one or several custom groups, based on their community tasks, access rights, or past activity. The list of user groups can be found at [The Team § User groups](/wiki/People/The_Team#user-groups).
+
+In addition to profile badges provided by group membership, users may have [custom forum titles](/wiki/User_title) displayed under the avatar in each of their posts.
```

Comment:
seems not very related to Forum. it's been a long time since any of these things were displayed or implemented as features of forums

Reply:
yes (I linked the wrong article) but also no (custom titles are still there: https://user-images.githubusercontent.com/4686101/144977675-d76fb1c3-19c6-4c6e-bc0b-0f7820cbdbe4.png)

Reply:
i dont see how that makes it any more related to forums than, for example, the country flag in same screenshot. it's user-related stuff that appears all over the site, so seems weird to explain it here as if it's specifically related to the forums

Reply:
while flags are self-explanatory, it would be interesting to know how many (and which) custom titles there are. I agree it isn't specific to the forums, but hey, neither are user groups, or BBCode, or moderation stuff, and the further you go on this road, the closer the article becomes to "well you can leave long-lasting messages, like these on other web sites, we also have several message categories, but they are all alike, end of story"